### PR TITLE
refactor(front/): Delete tailwindcss

### PR DESCRIPTION
**Pull request by MansourWolou**
Vuetify come not only with vue component to speed up developpement process but with css library( like bootstrap ) too. Tailwind is not need since we are already using vuetify.

Default css class created by vuetify are not needed too

Linked #47

### CODE REVIEW

**Review by MelvinCou — CHANGES_REQUESTED** [review]
Add this plugin for eslint : https://www.npmjs.com/package/eslint-plugin-vuetify

**Comment by MelvinCou** [comment]
> Add this plugin for eslint : https://www.npmjs.com/package/eslint-plugin-vuetify

This seems to look for deprecated features, nevermind.

File: front/src/App.vue [hunk 1]
```diff
@@ -1,108 +1,63 @@
...
+            <div class="mx-auto text-center">
+              <v-avatar color="brown">
+                <span class="text-h5">{{ user.initials }}</span>
+              </v-avatar>
+              <h3>{{ user.fullName }}</h3>
+              <p class="text-caption mt-1">
+                {{ user.email }}
+              </p>
+              <v-divider class="my-3"></v-divider>
+              <v-btn variant="text" rounded> Edit Account </v-btn>
```
Comment (Nokomi6) [inline front/src/App.vue:50]: No space for text in html

File: front/src/App.vue [hunk 2]
```diff
@@ -1,108 +1,63 @@
...
-    text-align: left;
-    margin-left: -1rem;
-    font-size: 1rem;
-
-    padding: 1rem 0;
-    margin-top: 1rem;
-  }
-}
-</style>
+<style scoped></style>
```
Comment (Nokomi6) [inline front/src/App.vue:63]: Remove if not used

File: front/src/components/__tests__/HomeView.spec.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import { describe, it, expect } from "vitest";
+
+import { mount, shallowMount } from "@vue/test-utils";
```
Comment (Nokomi6) [inline front/src/components/__tests__/HomeView.spec.ts:3]: Remove mount if not used

**Comment by MansourWolou** [comment]
> Add this plugin for eslint : https://www.npmjs.com/package/eslint-plugin-vuetify

don't need

**Review by MelvinCou — APPROVED** [review]

**Pull request merged**